julia-mode-latexsubs.el

```diff
@@ -1,3327 +1,6660 @@
+;;; julia-mode-latexsubs.el --- Generated variables for Julia mode -*- lexical-binding: t; -*-
 ;; This file is automatically generated. Do not edit by hand.
+
+(define-abbrev-table 'julia-latexsub-abbrev-table ()
+  "Abbrev table for Julia LaTeX substitution."
+  :regexp "\\(\\\\[[:graph:]]+\\)")
+
 (defconst julia-mode-latexsubs
   (let ((table (make-hash-table :test 'equal)))
     (puthash "\\exclamdown" "¡" table)
+    (define-abbrev julia-latexsub-abbrev-table "\\exclamdown" "¡" nil :system t)
```

Comment:
I'm curious, is there any particular reason why each hash entry is duplicated for the abbrev table instead of leaving the hash table alone and then looping through the entries to create the abbrev table at the end?

Reply:
There isn't. I just found this the simplest way of writing it.